apps/chat/src/components/Chat/ChatDropArea.tsx

```diff
@@ -0,0 +1,42 @@
+import { ReactNode } from 'react';
+
+import { useUploadFilesHandler } from '@/src/hooks/useUploadFilesHandler';
+
+import { getQuickAttachmentsSavingPath } from '@/src/utils/app/conversation';
+
+import { ConversationsSelectors } from '@/src/store/conversations/conversations.reducers';
+import { useAppSelector } from '@/src/store/hooks';
+
+import { FileDropArea } from '@/src/components/Files/FIleDropArea';
+
+interface ChatDropAreaProps {
+  children: ReactNode;
+}
+
+export const ChatDropArea = ({ children }: ChatDropAreaProps) => {
+  const availableAttachmentsTypes = useAppSelector(
+    ConversationsSelectors.selectAvailableAttachmentsTypes,
+  );
+  const maximumAttachmentsAmount = useAppSelector(
+    ConversationsSelectors.selectMaximumAttachmentsAmount,
+  );
+  const canAttachFiles = useAppSelector(
+    ConversationsSelectors.selectCanAttachFile,
+  );
+
+  const handleUploadFiles = useUploadFilesHandler(
+    getQuickAttachmentsSavingPath(),
+    maximumAttachmentsAmount,
+    availableAttachmentsTypes,
+  );
```

Comment:
I see this code block with selectors and custom hook at least twice - maybe create additional custom hook `useChatUploadFiles` instead 
```suggestion
  const { handleUploadFiles, canAttachFiles } = useChatUploadFiles();
```

Reply:
`useChatUploadFiles()` with default param `path = getQuickAttachmentsSavingPath()`. If we will want to replace the path